x-pack/plugins/enterprise_search/server/lib/indices/fetch_indices.test.ts

```diff
@@ -417,14 +423,13 @@ describe('fetchIndices lib function', () => {
           'alias-search-prefixed-regular-index',
           'search-alias-search-prefixed-regular-index',
           '.ent-search-engine-documents-12345',
-          'alias-.ent-search-engine-documents-12345',
           'search-alias-.ent-search-engine-documents-12345',
-          'search-prefixed-.ent-search-engine-documents-12345',
-          'alias-search-prefixed-.ent-search-engine-documents-12345',
           'search-alias-search-prefixed-.ent-search-engine-documents-12345',
           'search-alias-hidden-index',
           'search-alias-search-prefixed-hidden-index',
-        ].map(getIndexReturnValue)
+        ]
+          .map(getIndexReturnValue)
+          .sort(sortIndices)
       );
 
       // This is the list of mock indices and aliases that are:
```

Comment:
I may be a bit overcautious, but as it is another layer of filtering, I would add the deleted values above to the expect block just below

`alias-.ent-search-engine-documents-12345`
`search-prefixed-.ent-search-engine-documents-12345`
`alias-search-prefixed-.ent-search-engine-documents-12345`